examples/examples.scss

```diff
@@ -0,0 +1,287 @@
+@import "../../src/main.scss";
```

Comment:
That's a wonderful idea! I would be really excited to see that PR. Style linting can wait for that pr in that case